helix-control-m/2-external-monitoring-tools-examples/alerts-to-helix-itsm/README.md

```diff
@@ -86,3 +87,20 @@ BMC Helix Remedy ITSM or on-prem with REST APIs enabled.
   * will compose a URL for the case to display, if the alert is that of a job.
   * There is a feature for those that want to send an email.
     * If you want to send an email and not the ticket, you will need to comment all lines that send the ticket and ensure variables set at the return are not used in messages later.
+
+## How to use it
+
+### Installation
+
+* Clone this repository
+
+* Install the Automation API CLI in the system you will use to receive the alerts and process them to forward to Helix ITSM.
```

Comment:
I'd suggest revising to: "...receive the alerts and forward to Helix ITSM. "

Reply:
It is not just forward, but to do some processing (evaluate, filter, augment data). Open to suggestions to keep improving, after the initial merge to resolve bugs.

Reply:
"...you will use to receive and process alerts before forwarding them to Helix ITSM."

Reply:
I will change use to employ. 

Install the Automation API CLI in the system you will employ to receive the alerts and process them to forward to Helix ITSM.